pango/Gir.toml

```diff
@@ -206,6 +203,14 @@ boxed_inline = true
 name = "Pango.Context"
 status = "generate"
     [[object.function]]
+    name = "new"
+        [object.function.return]
+        nullable = false
+    [[object.function]]
+    name = "get_language"
+        [object.function.return]
+        nullable = true
```

Comment:
Add links here to your MR that makes the overrides unnecessary

Reply:
Good idea!